From 4a1f066131d96c349934681313b3ed622120ba32 Mon Sep 17 00:00:00 2001
From: Tony Cheng <tony.cheng@amd.com>
Date: Mon, 17 Oct 2016 12:05:55 -0400
Subject: [PATCH 1343/1722] drm/amd/dal: DCE 11 only support 3 audio.

- working in the past as PORT_CONNECTIVITY blocks un-supported audio
 endpoints.
- no need to create audio then destroy it after finding out end point isn't
 supported.

Change-Id: I4dd7ff722dbf1f69e5bbc17b567044c5e7444bd2
Signed-off-by: Tony Cheng <tony.cheng@amd.com>
Acked-by: Harry Wentland <harry.wentland@amd.com>
---
 drivers/gpu/drm/amd/dal/dc/dce110/dce110_resource.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/dal/dc/dce110/dce110_resource.c b/drivers/gpu/drm/amd/dal/dc/dce110/dce110_resource.c
index 3d48dfc..a70135c 100644
--- a/drivers/gpu/drm/amd/dal/dc/dce110/dce110_resource.c
+++ b/drivers/gpu/drm/amd/dal/dc/dce110/dce110_resource.c
@@ -314,7 +314,7 @@ static const struct dce110_clk_src_reg_offsets dce110_clk_src_reg_offsets[] = {
 
 
 static const struct resource_caps res_cap = {
-	.num_audio = 4,
+	.num_audio = 3,
 };
 
 #define CTX  ctx
-- 
2.7.4

